app/components/pullRequestPanel/pull.request.panel.component.html

```diff
@@ -22,8 +22,11 @@
     <!-- Nav bar on the right of the screen to show/hide PR Icon. -->
     <div class="right-pr-nav">
         <i (click)="togglePRPanel()"><img src="./assets/PullRequestIcon.svg"></i>
+        <hr>
+        <button class="btn btn-primary center-block" height="40" width="48" id ="unpushed" font-weight="bold" title="number of un-pushed commits">  </button>
```

Comment:
- I don't love this being a button, hovering over it leads me to believe this is clickable which it isn't. Perhaps using a badge might be better https://getbootstrap.com/docs/3.4/components/#badges
- It also isn't immediately obvious what this number means until you hover over it which I suppose is okay, maybe moving it to near the commit panel or the push button might be more clear?

Reply:
Thanks, @mehrler12 A badge also used the button tag that we used as per your reference. A button tag becomes clickable when you include an onclick() function which we don't use. 
-Also, James approved the position and the current UI because the number would affect how the commit panel is displayed.

Reply:
Also when you open a fresh repo, the number is not -1. The number only updates if there are changes on a repo locally that has not been pushed.

Reply:
![image](https://user-images.githubusercontent.com/50643496/59315069-fe298000-8c6c-11e9-94a4-44f7abaa033c.png)
Okay, no worries on the placement then! 
Badges use the span tag, not the button tag. I realize that the button is only clickable when you set onClick but it still looks like it's clickable without it (Changing the pointer and colour) which is where my concern comes from.

Reply:
@aeogunmuyiwa Maybe we could change the background colour of the badge to white so it looks less clickable?

Reply:
@mehrler12 Thanks for that catch, I fixed the variable subtraction. If you could pull the branch and test it again that would be awesome. @trstndvll  I changed the background color to white now. thanks.

Reply:
Still -1 
![image](https://user-images.githubusercontent.com/50643496/59316848-4d73ae80-8c75-11e9-9c33-a0a798486a22.png)
I'll give you access to my dummy repo, maybe it's something in there?
As per the button I was fine with the colour before, just that it changes on hover but if it's too much to change that's alright how it is! I'll submit my approval

Reply:
Continuing on from @mehrler12, and with the newest changes, it still acts like a button. While changing the color was a good idea, on mouse over it still 'bezels'/highlights and makes it seem like a button. Is there a way to stop that from happening @aeogunmuyiwa?

Reply:
There’s no such functionality like acts like a button . James wanted it look like a badge and that’s a badge. It’s only a button if it becomes clickable (onclick ) function which it doesn’t have . We used the hover to give it a context . Thanks @Athenasko

Reply:
> Still -1

@mehrler12 Interesting! I'm on a Mac and I'm not even able to get into a new repo to test this out. I go to make one and it just spins endlessly at "loading graph." It's the same story on current version of Master. Is creating a local repo working as expected for you?

Reply:
@trstndvll I don’t think that issue is related to this ticket though . It’s has something to do with master I suppose

Reply:
@trstndvll I added you to a private repo I've been using for testing try cloning that?